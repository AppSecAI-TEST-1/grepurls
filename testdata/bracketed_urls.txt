[http://www.google.com]
<http://www.wiki.org>
{mailto:divye@divye.me}
"ftp://divye:password@fake.me/"
'mailto:divye@divye.me'
'http://de.wikipedia.org/wiki/Uniform_Resource_Identifier"
(ftp://ftp.is.co.za/rfc/rfc1808.txt)
{file:///C:/Users/Benutzer/Desktop/Uniform%20Resource%20Identifier.html)
[file:///etc/fstab}
<geo:48.33,14.122;u=22.5'
