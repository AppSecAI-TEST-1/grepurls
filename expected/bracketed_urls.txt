http://www.google.com
http://www.wiki.org
mailto:divye@divye.me
ftp://divye:password@fake.me/
mailto:divye@divye.me
ftp://ftp.is.co.za/rfc/rfc1808.txt
